src/Workspaces/CSharp/Portable/CodeGeneration/CSharpSyntaxGenerator.cs

```diff
@@ -3531,6 +3531,11 @@ public override SyntaxNode ExpressionStatement(SyntaxNode expression)
             return SyntaxFactory.ExpressionStatement((ExpressionSyntax)expression);
         }
 
+        public override SyntaxNode ConditionalAccessExpression(SyntaxNode expression, SyntaxNode whenNotNull)
+        {
+            return SyntaxFactory.ConditionalAccessExpression(Parenthesize(expression), SyntaxFactory.MemberBindingExpression((SimpleNameSyntax)whenNotNull));
```

Comment:
1. Yes, I would prefer that too. Is there a maximum length agreed on somewhere?
2. Thanks for explaining. The thing is: the extra factory method that is needed varies for C# (`SyntaxFactory.MemberBindingExpression`) and VB (`SyntaxFactory.SimpleMemberAccessExpression`). How would that factory method be named then, or should it not be in the language-independent interface?